### PR TITLE
add AnchorJS

### PR DESCRIPTION
**Pull request by afeld**
As [requested](https://18f.slack.com/archives/devops/p1453859685000134) by @adelevie.

http://bryanbraun.github.io/anchorjs/

This will enable it for every Guide. Thoughts about going this direction, versus adding it to [the template](https://github.com/18F/guides-template), where it can be removed/customized by the user?

### CODE REVIEW

File: assets/js/18f-guide.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  $('.sidebar-nav').each(function() {
+    accordion($(this));
+  });
+});
+
+// http://bryanbraun.github.io/anchorjs/
+anchors.options = {
+  visible: 'touch'
+};
+anchors.add('h3,h4,h5');
```
Comment (mbland) [inline assets/js/18f-guide.js]: This didn't do much in 18F/guides-template until I added `h2` to the mix, but started looking pretty _sweeet_ when I did. Any reason not to add it?

File: assets/js/18f-guide.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$(function() {
+  $('.sidebar-nav').each(function() {
+    accordion($(this));
+  });
+});
```
Comment (mbland) [inline assets/js/18f-guide.js]: Why hoist this out from the original `accordion.js` file?

**Comment by mbland** [comment]
Whoops, looks like merging #39 first caused this and #38 to conflict.

File: assets/js/18f-guide.js (hunk 2 above)
Comment (afeld) [inline assets/js/18f-guide.js]: Ideally (in my opinion) the off-the-shelf Web Design Standards could be used, so was trying to move the application-specific code out of `accordion.js`. That file still has your tweak to the `Accordion` constructor, though.

File: assets/js/18f-guide.js (hunk 1 above)
Comment (afeld) [inline assets/js/18f-guide.js]: It didn't seem that useful to link to the page title, which is an `<h2>`.

https://github.com/18F/guides-style/blob/51a8eb38d6397b076a697607fd2d8aae5e39cd58/lib/guides_style_18f/layouts/default.html#L26

My thinking was that all of the sub-sections below should be `<h3>`s, `<h4>`s, etc., but I don't feel super strongly about it.

File: assets/js/18f-guide.js (hunk 2 above)
Comment (mbland) [inline assets/js/18f-guide.js]: Gotcha. It's cool to hoist out like this.

File: assets/js/18f-guide.js (hunk 1 above)
Comment (mbland) [inline assets/js/18f-guide.js]: That makes sense. The "right thing" is to make sure the page title is the only `<h2>`. Let's keep `h2` of of this, then.

**Pull request merged**